tests/attacks/property_inference_test.py

```diff
@@ -9,18 +9,19 @@
 
 
 def test_property_inference_attack():
-    train_dataset, test_dataset = dataset_downloader("CIFAR10")
+    train_dataset, test_dataset = dataset_downloader("MNIST")
     input_shape = test_dataset[0][0].shape
-    num_elements_per_classes = {0: 1000, 1: 1000}
+    num_elements_per_classes = {0: 500, 1: 500}
     num_classes = len(num_elements_per_classes)
 
     train_set = new_dataset_from_size_dict(train_dataset, num_elements_per_classes)
 
     model = ConvNet(num_classes, input_shape)
-    trainer(train_set, num_elements_per_classes, model)
+    print("Start training target model ...\n")
+    trainer(train_set, num_elements_per_classes, model, num_epochs=2, verbose=2)
```

Comment:
How I understood the test it's an intern function for pytest. So in my point of view a verbose of 2 or 1 isn't necessary

Reply:
Will delete it then